packages/website/src/content/02-04-web-widget-try-live.tsx

```diff
@@ -97,7 +98,11 @@ export const WebWidgetTryLive = () => {
         <TitleBarEditor
           value={titleBar}
           onChange={(val: any) => {
-            setTitleBar((prev) => ({ ...prev, ...val }));
+            setTitleBar((prev) => {
+              const titleBar = { ...prev, ...val };
+              if (titleBar.logo?.length === 0) return omit(["logo"], titleBar);
```

Comment:
small behavior change on the website: `logo` field is removed from the snippet if it's an empty string